MPhys/Direct import/python input.ipynb

```diff
@@ -2,7 +2,11 @@
  "cells": [
   {
    "cell_type": "code",
+<<<<<<< HEAD
```

Comment:
Still some merge conflict issues left in this file; need to fix these.  (Note: I have got VSCode running via WSL2 - remarkably straightforward - so merging in the future shouldn't be as much of an issue, hopefully).